server/src/main/kotlin/org/elaastic/assignment/AssignmentService.kt

```diff
@@ -207,6 +207,9 @@ class AssignmentService(
     fun getRegisteredUsers(assignment: Assignment): List<LearnerAssignment>
         = learnerAssignmentRepository.findAllByAssignment(assignment)
 
+    fun getRegisteredUser(assignment: Assignment, user: User): LearnerAssignment?
```

Comment:
I do not understand the name of this function. A `LearnerAssignment` is not the registered user.

Reply:
I just take the name of the above function and put it in singular.

With this function, I want to retrieve the `LearnerAssignment` associated with the given `User` in the given `Assingment`. So in other words I want the user who is registered in the given assignement.

Reply:
Oh, my bad. I read `LearnerAssignment` and I thought to `LearnerSequence`.

So, you're totally right to associate `LearnerAssignment` witht the concept of "registered user".
I'm just wondering if it wouldn't be better to use a single name for that concept. It's not a big deal and it can stay like this.